core/src/franchise/player/player.resolver.ts

```diff
@@ -238,6 +238,67 @@ export class PlayerResolver {
         return "SUCCESS";
     }
 
+    @Mutation(() => String)
+    @UseGuards(GqlJwtGuard, MLEOrganizationTeamGuard([MLE_OrganizationTeam.MLEDB_ADMIN, MLE_OrganizationTeam.LEAGUE_OPERATIONS]))
+    async changePlayerElo(
+        @Args("playerId", {type: () => Int}) playerId: number,
+        @Args("salary", {type: () => Float}) salary: number,
+        @Args("elo", {type: () => Float}) elo: number,
+    ): Promise<string> {
+        const player = await this.playerService.getPlayer({
+            where: {id: playerId},
+            relations: {
+                member: {
+                    user: {
+                        authenticationAccounts: true,
+                    },
+                    organization: true,
+                    profile: true,
+                },
+                skillGroup: {
+                    id: true,
+                    organization: true,
+                    game: true,
+                    profile: true,
+                },
+            },
+        });
+
+        const inputData: ManualEloChange = {
+            id: playerId,
+            salary: salary,
+            elo: elo,
+        };
+
+        await this.playerService.mle_movePlayerToLeague(playerId, salary, player.skillGroupId);
+        await this.playerService.updatePlayerStanding(playerId, salary, player.skillGroupId);
+        await this.eloConnectorService.createJob(EloEndpoint.EloChange, inputData);
+
+        return "SUCCESS";
+    }
+    
+    @Mutation(() => [Player])
+    @UseGuards(GqlJwtGuard, MLEOrganizationTeamGuard([MLE_OrganizationTeam.MLEDB_ADMIN, MLE_OrganizationTeam.LEAGUE_OPERATIONS]))
+    async changePlayerEloBulk(@Args("files", {type: () => [GraphQLUpload]}) files: Array<Promise<FileUpload>>): Promise<string> {
+        const csvs = await Promise.all(files.map(async f => f.then(async _f => readToString(_f.createReadStream()))));
+
+        const results = csvs.flatMap(csv => csv.split(/(?:\r)?\n/g).map(l => l.trim().split(","))).filter(r => r.length > 1);
+        const parsed = EloRedistributionSchema.parse(results);
+        
+        let numFailed = 0;
+        
+        for (const player of parsed) {
+            try {
+                await this.changePlayerElo(player.playerId, player.salary, player.newElo);
+            } catch {
+                numFailed++;
+                continue;
+            }
+        }
+        
+        return (numFailed === 0) ? `Success, all elos changed.` : `${numFailed} elos were unable to be changed.`;
```

Comment:
Return a list of who failed, not just a number so we can look at who/why and not need to guess.

Reply:
Done.